# Tray contents message
# This structure contains the information of the current contents of a tray.

# The ID of the tray
string tray

# Collection of objects
DetectedProduct[] objects
